content/python/metric_tracking_object.py.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Working with Metric_Tracking_Object.py"
+description: "A few examples on interacting with Metric tracking object"
+
+date: "2019-12-20"
+classes: 
+    - "SoftLayer_Metric_Tracking_Object"    
+tags:
+    - "metric_tracking"
+---
+
+[Metric_Tracking_Object](https://sldn.softlayer.com/reference/services/SoftLayer_Metric_Tracking_Object/)
+
+### Get the all metric tracking ids
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+virtual_service = client['Account']
+
+mask = 'id,metricTrackingObject[id]'
+
+try:
+    response = virtual_service.getVirtualGuests(mask = mask)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Get back bone bandwidth graph
+
+```python
+import SoftLayer
+import json
+
+
+client = SoftLayer.create_client_from_env()
+virtual_service = client['Metric_Tracking_Object']
+
+body = 'HostReservedMemoryUsage'
+"""
+Set with Metric_Tracking_Object id. 
+"""
+id = '123456789'
+try:
+    response = virtual_service.getBackboneBandwidthGraph(body, id = id)
```

Comment:
This API is actually being removed /  deprecated. https://sldn.softlayer.com/release_notes/2019/20191216/
They will technically still work until NETWORK-10937 is published, but for now lets just remove this one example.